shuup/gdpr/static_src/gdpr.js

```diff
@@ -40,7 +40,7 @@
             url: $("#consent-form").attr("action"),
             type: 'POST',
             data: data,
-            success: function() {
+            complete: function() {
```

Comment:
why complete? if the request fails, that things shouldn't be removed

Reply:
"gdpr consent can be accepted on any page where jquery is imported(before it was only available upon successful request completion)," for instance, 404 page wasn't failing just because it didn't have jquery imported, when jquery was imported it would return an "error" 404 request instead of a successful 302 or 200. This was blocking the code execution for removing GDPR boxes.

Reply:
IMO, you should warn user when an error occurs and then remove the preferences or something.. this is hiding the actual error